app/models/pipeline_run.rb

```diff
@@ -814,9 +883,15 @@ def check_and_enqueue(output_state)
     if output_ready?(output)
       output_state.update(state: STATUS_LOADING_QUEUED)
       Resque.enqueue(ResultMonitorLoader, id, output)
+    # check if job is done more than a minute ago
     elsif finalized? && pipeline_run_stages.order(:step_number).last.updated_at < 1.minute.ago
-      # check if job is done more than a minute ago
-      output_state.update(state: STATUS_FAILED)
+      # HACK: don't fail for missing insert, size metrics, they are optional
```

Comment:
Great PR, but would you mind finding a different mechanism for making insert size loading failures non-blocking? I think you could have a rescue block in `db_load_insert_size_metrics` and in that block you should at the very least send an alert about the failure